README.md

```diff
@@ -3,79 +3,120 @@
-   - `OIDC_MEDEWERKER_IDENTIFICATIE_CLAIM` and `OIDC_MEDEWERKER_IDENTIFICATIE_TRUNCATE`: the current ZGW standards limit the medewerker identificatie to a maximum of 24 characters. If you are using KISS with OpenKlant / OpenZaak, this means you either need to configure a claim in `OIDC_MEDEWERKER_IDENTIFICATIE_CLAIM` that will never exceed this limit, or configure for it to be truncated by setting `OIDC_MEDEWERKER_IDENTIFICATIE_TRUNCATE` to 24. If you are using KISS with the eSuite, make sure you configure a claim in `OIDC_MEDEWERKER_IDENTIFICATIE_CLAIM` that contains te username known in the eSuite.
-   - `KVK_BASE_URL`: for the KvK test environment, use `https://api.kvk.nl/test/api` 
-   - `KVK_API_KEY`: for the KvK test environment, look for the API key on [the KvK website](https://developers.kvk.nl/documentation/testing)
-2. Download the Root and intermediate certificates from [the KvK website](https://developers.kvk.nl/documentation/install-tls-certificate#download-certificates) and place them in a `certificates` folder in the root of the repo
-Now, you can either run the application from Visual Studio or with docker-compose
-
-## Run from Visual Studio 2022 
-Be sure to set-up environment variables first
-1. Make sure you've installed Docker Desktop version 4.19.0 or newer (preferably with WSL2 if using windows) and visual studio version 17.5.5 or newer
-2. Open KISS-frontend.sln in Visual Studio 2022
-3. Right-click the solution in the Solution Explorer and pick Configure Startup Projects
-4. Select Multiple startup projects, and set the Action to Start for docker-compose and KISS-frontend
-5. Startup the solution and wait for both the BFF and the frontend to be ready (note, initially you might get an error, due to different startup times of individual components. refresh the page after a few moments)
+## Setup environment variables
+
+1. Make a copy of `.env.local.example`, rename it to: `.env.local` and fill in the required secrets.
+
+   NOTE: The example file does not contain all environment variables. A complete overview of all possible variables can be found at: https://kiss-klantinteractie-servicesysteem.readthedocs.io/en/latest/installation/configuratie.html
+
+1. Download the `Root CA` and `Intermediate CA` certificates from [the KvK website](https://developers.kvk.nl/documentation/certificates) and place them in a `certificates` folder in the root of the repo.
```

Comment:
wat mij betreft halen we het stuk over de certificaten eruit.

kopie uit slack:
```
I've double checked. We used to need those certificates, otherwise we couldn't send any requests to the kvk api. These days, that's not necessary anymore. We can:
remove the certificates from the repo
remove any reference to the certificates in docker compose files and the Dockerfile
remove the instructions from the readme
It would also be good to test this on the test environment
```